src/demo-app/a11y/toolbar/toolbar-a11y.html

```diff
@@ -0,0 +1,26 @@
+<div class="demo-toolbar">
+  <section>
+    <h2>Basic Toolbar with Text  (e.g. Only display app’s name)</h2>
+    <md-toolbar>My App</md-toolbar>
```

Comment:
A toolbar by itself isn't really representative of how the component will be used. It typically acts as a header for some content. In this capacity, it should either use `role="heading"` and `aria-level` OR it should _contain_ a header element (`h1` -> `h6`). 

The examples should represent a real use-case as much as possible.